notebooks/mllib/Planes.snb

```diff
@@ -59,7 +59,43 @@
   }, {
     "metadata" : { },
     "cell_type" : "markdown",
-    "source" : "For this small example, we're going to use a subset of the data, that is, only the year [2008](http://stat-computing.org/dataexpo/2009/2008.csv.bz2).\n\n**Warning**: this data is distributed in bz2.\n\nData can be found [here](http://stat-computing.org/dataexpo/2009/the-data.html)"
+    "source" : "For this small example, we're going to use a subset of the data, that is, only the year [2008](http://stat-computing.org/dataexpo/2009/2008.csv.bz2). Data for all available years can be found [here](http://stat-computing.org/dataexpo/2009/the-data.html).\n\n**Warning**: this data is distributed in bz2."
+  }, {
+    "metadata" : { },
+    "cell_type" : "markdown",
+    "source" : "Create directory for our data to live in."
+  }, {
+    "metadata" : {
+      "trusted" : true,
+      "collapsed" : false
+    },
+    "cell_type" : "code",
+    "source" : ":sh mkdir -p /home/noootsab/data/dataexpo_2009",
```

Comment:
maybe using `/tmp`?

Which won't work on windows, but there is a PR to change all notebooks to be platform indep.  so we'll work on them all with the same pattern at that time